reporting/impl/src/main/resources/org/jboss/windup/reporting/rules/generation/techreport/techReport-hierarchy.tags.xml

```diff
@@ -99,8 +98,9 @@
             <tag name="techBox:webservice" title="WebService" pseudo="true" parents="techRow:embedded" icon="Global_Ecosystem">
                 <tag ref="webservices"/>
             </tag>
-            <tag name="techBox:rest" title="REST" pseudo="true" parents="techRow:embedded" icon="Science_Circuit">
+            <tag name="techBox:rest" title="REST" pseudo="true" parents="techRow:embedded, techRow:java-ee" icon="Science_Circuit">
```

Comment:
I see this is adding java-ee as parent for jax-rs, but also for rest that before it didnt have this parent.
Is this a wanted effect ?

Reply:
Yes, we want in the case of Jersey for instance, for the count to go in 'Embedded/REST' but for Jax-RS, for it to go in JAVAEE/REST